pydocx/openxml/wordprocessing/picture.py

```diff
@@ -6,10 +6,10 @@
 )
 
 from pydocx.models import XmlModel, XmlCollection
-from pydocx.openxml.vml import Shape
+from pydocx.openxml.vml import Shape, Rect
 
 
 class Picture(XmlModel):
     XML_TAG = 'pict'
 
-    children = XmlCollection(Shape)
+    children = XmlCollection(Shape, Rect)
```

Comment:
So, as a noob trying to understand, we're looking at something like this https://msdn.microsoft.com/en-us/library/documentformat.openxml.wordprocessing.picture(v=office.15).aspx which states all the various VML shapes can sit under it?